src/core_plugins/elasticsearch/index.js

```diff
@@ -42,15 +71,24 @@ module.exports = function ({ Plugin }) {
           esRequestTimeout: options.requestTimeout,
           esShardTimeout: options.shardTimeout,
           esApiVersion: options.apiVersion,
+          esDataIsTribe: options.tribe ? true : false,
         };
       }
     },
 
     init(server, options) {
       const kibanaIndex = server.config().get('kibana.index');
+      const clusters = createClusters(server);
+
+      server.expose('getCluster', clusters.get);
+      server.expose('createCluster', clusters.create);
+
+      server.expose('filterHeaders', filterHeaders);
+      server.expose('ElasticsearchClientLogging', clientLogger(server));
+
+      createDataCluster(server);
+      createAdminCluster(server);
```

Comment:
It wasn't necessary for testing to work but it isolated the code the made it easier to test.

Both of these call `server.plugins.elasticsearch.createCluster()` which expose the data and admin clusters to the rest of the application. They are then accessible via `server.plugins.elasticsearch.getCluster()`